modules/flowable-ui-modeler/flowable-ui-modeler-app/oryx-editor-code/scripts/Core/StencilSet/stencilset.js

```diff
@@ -399,6 +399,113 @@ ORYX.Core.StencilSet.StencilSet = Clazz.extend({
 		
 		var defaultPosition = 0;
 		
+    
+    if(!customTaskConfigs) {
+      new Ajax.Request(
+          FLOWABLE.CONFIG.contextRoot + "/app/workflow/model/taskconfigs", {
+            asynchronous:false, method:'get',
+            onSuccess:function(result) {
+              customTaskConfigs=result.responseText.evalJSON();
+              },
+            onFailure:function(result) {}
+        });           
+    }
+    
+    // start added by simon to handle custom task 
```

Comment:
is it worth adding a custom string here to make such stuff easier to find eg
// rds - start added bysimon to handle custom task